### PR TITLE
feat: create a snapshot workflow

### PR DESCRIPTION
**Pull request by grablack**

### CODE REVIEW

File: .github/workflows/test-release.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
```
Comment (merlinpaypal) [inline .github/workflows/test-release.yml]: Use `microsoft` here and in `test_instrumentation`

File: build.gradle [hunk 2]
```diff
@@ -5,8 +5,8 @@ import org.jmailen.gradle.kotlinter.tasks.FormatTask
 buildscript {
 	ext.modules = [
 		"sdkVersionCode"      : 10000,
-		"sdkVersionName"			: "1.0.0-alpha.00-SNAPSHOT",
-		"androidMinSdkVersion": 23,
+		"sdkVersionName"      : "1.0.0-alpha.00",
+		"androidMinSdkVersion": 21,
```
Comment (merlinpaypal) [inline build.gradle]: Why the switch to 21? The PayPal Android SDK uses 23+.
https://github.com/paypal/paypal-android?tab=readme-ov-file#support

**Review by merlinpaypal — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (grablack) [reply build.gradle]: The main [gradle is set to 21](https://github.com/paypal/paypal-android/blob/main/build.gradle). [PayPalNativePayments](https://github.com/paypal/paypal-android/blob/main/PayPalNativePayments/build.gradle) has it set to 23 and the[ demo xml is overriding the project](https://github.com/paypal/paypal-android/blob/main/Demo/src/main/AndroidManifest.xml#L7) to use that. I believe that is why its listed as 23+. Dependencies use the gradle version so it wants 21. Adding messaging works when I set it to 21. I get this error when adding messaging at sdk version 23 : 
```
Manifest merger failed : uses-sdk:minSdkVersion 21 cannot be smaller than version 23 declared in library [com.paypal.messages:paypal-messages...] as the library might be using APIs not available in 21
	Suggestion: use a compatible library with a minSdk of at most 21,
		or increase this project's minSdk version to at least 23
```

File: .github/workflows/test-release.yml (hunk 1 above)
Comment (grablack) [reply .github/workflows/test-release.yml]: changed

File: build.gradle (hunk 2 above)
Comment (grablack) [reply build.gradle]: set to 23

**Review by merlinpaypal — APPROVED** [review]

**Pull request merged**